Starboard: Continuous Updates #5

choccy4 opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->
## Short description
This PR is for changes made to Starboard, the station. Anyone who wants can submit changes and i will add them, there is a post open in the suggestions tab on discord!
<!-- What do you propose to change with your PR? -->

## Why we need to add this
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
Because I am not absolutely cracked at map making, I make mistakes!

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->
![image](https://github.com/user-attachments/assets/2ea16da8-cb9f-4a04-80c9-964149f5e4c9)
![image](https://github.com/user-attachments/assets/bc9dbce7-319a-4706-87ad-ea79f9ed7549)
![image](https://github.com/user-attachments/assets/e98f4e77-6904-47ac-8835-5b056a670a23)
![image](https://github.com/user-attachments/assets/3f86d098-8936-4c0c-abce-9dbecb840960)
![Starboard-1305](https://github.com/user-attachments/assets/f92e8ca2-969d-493c-a8b1-8b7d6816df5b)
## Checks
<!-- check boxes for faster reviewing of your PR -->

- [X] I do not require assistance to complete the PR.
- [X] Before posting/requesting review of a PR, I have verified that the changes work.
- [X] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [X] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.

:cl: STARLIGHT TEAM
- add: Added Starlight.
- remove: Removed SS13.
- tweak: Changed SS14.
- fix: Fixed Rinary.
-->
:cl: choccy
- fix: (Starboard) Roboticists are real
- tweak: (Starboard) Two wardens?
- tweak: (Starboard) Many small details changed, from decals to doors and decor
- add: (Starboard) Lots of new maints areas

choccy4 committed:
added roboticists, heinously added two wardens

choccy4 committed:
small fixes and tweaks

choccy4 committed:
Merge branch 'ss14Starlight:Starlight' into starboard-2

PubliclyExecutedPig commented:
two wardens? provide a screenshot of any map changes made by this

choccy4 commented:
> two wardens? provide a screenshot of any map changes made by this

okie dokie

choccy4 commented:
> two wardens? provide a screenshot of any map changes made by this

bruzz, i added screenshots of the stuff changed on the map, primarily cap suit storage, stove, oven and general cam router - i also changed the proto to have the new roles

PubliclyExecutedPig commented:
> > two wardens? provide a screenshot of any map changes made by this
> 
> bruzz, i added screenshots of the stuff changed on the map, primarily cap suit storage, stove, oven and general cam router - i also changed the proto to have the new roles

they didnt load for me when i looked

choccy4 commented:
> > > two wardens? provide a screenshot of any map changes made by this
> > 
> > 
> > bruzz, i added screenshots of the stuff changed on the map, primarily cap suit storage, stove, oven and general cam router - i also changed the proto to have the new roles
> 
> they didnt load for me when i looked

theyre new, thats why!

choccy4 committed:
Merge branch 'ss14Starlight:Starlight' into starboard-2

choccy4 committed:
reduced max chems 3 to 2

choccy4 committed:
added lots of new maints, minor misc tweaks

choccy4 committed:
minpop is now 90

choccy4 committed:
adds new maints, other misc tweaks

choccy4 committed:
Merge branch 'ss14Starlight:Starlight' into starboard-2
